tests/resources/deeprec/movielens/network_xdeepFM.yaml

```diff
@@ -0,0 +1,61 @@
+#data
```

Comment:
is it possible to eliminate the yaml file and add the information directly to the notebook? 

To upload to github a yaml file is tricky because if you want to do experiments and change the inputs, we need to modify this file, which gets tracked. 

I had this same conversation with Jianxun and finally he download a yaml file from a blob in the notebook and then set some parameters directly. Is it possible to just set the parameters in the notebook?